app/src/main/java/com/github/pockethub/android/core/PageIterator.java

```diff
@@ -56,11 +57,11 @@ public void remove() {
 
     @Override
     public List<V> next() {
-        if(!this.hasNext()) {
+        if (!this.hasNext()) {
```

Comment:
I'm not sure whether I'm just being nit picky, but does such a styling change reflect the intent of the PR?